app/medInria/medRegistrationWorkspace.cpp

```diff
@@ -45,31 +45,14 @@ class medRegistrationWorkspacePrivate
 
 medRegistrationWorkspace::medRegistrationWorkspace(QWidget *parent) : medAbstractWorkspace(parent), d(new medRegistrationWorkspacePrivate)
```

Comment:
Can't you make it a medSelectorWorkspace and in some way force the selectorTbx to be a registrationSelectorToolBox?
I think @fcollot had an idea for this.